src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,24 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.figures.Figure;
+
 public class HelloWorld {
+    public static final int FIGURE_COUNT = 6;
 
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[FIGURE_COUNT];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        //fill first half of array
+        for (int i = 0; i < FIGURE_COUNT / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        //fill second half of array
+        for (int i = FIGURE_COUNT / 2; i < FIGURE_COUNT; i++) {
+            figures[i] = figureSupplier.getDefaultFigure();
+        }
```

Comment:
you can do it using one loop and one if-else statement

Reply:
Not resolved